plugins/optimization-detective/class-od-link-collection.php

```diff
@@ -194,7 +194,7 @@ static function ( array $carry, array $link ): array {
 					&&
 					is_int( $last_link['maximum_viewport_width'] )
 					&&
-					$last_link['maximum_viewport_width'] + 1 === $link['minimum_viewport_width']
+					$last_link['maximum_viewport_width'] === $link['minimum_viewport_width']
```

Comment:
Key change. The addition of 1 is no longer needed because the previous maximum is intended to equal the next minimum.